examples/nek5000/run-nek-example.sh

```diff
@@ -31,7 +31,7 @@ fi
 # Set defaults for the parameters
 : ${CEED_DIR:=`cd ../../; pwd`}
 nek_examples=(bp1 bp3)
-nek_spec=/cpu/self
+nek_spec=/cpu/self/ref
```

Comment:
`/cpu/self/ref` isn't the recommended CPU backend. I'd prefer this to be set to `/cpu/self/blocked` or `/cpu/self` (which defaults to `/cpu/self/blocked`)